tools/PI/DevHome.PI/Controls/AddToolControl.xaml.cs

```diff
@@ -45,8 +99,6 @@ private void HandleBrowseButton()
                     lpstrFilter = pFilter,
                     nFilterIndex = 1,
                     nMaxFile = 255,
-
-                    // TODO - This should be the Settings window, not the bar window
```

Comment:
>// TODO  [](http://example.com/codeflow?start=20&length=8)

Is this not valid anymore?

Reply:
Ahh... this is because there isn't a separate Settings window anymore. Got it.

Reply:
Settings hasn't been a free-floating window for a while now, and the Settings page is within the bar window.